### ISSUE TITLE
Confusing notation in Lesson 6.4

### ISSUE THREAD

**Issue by lnson** [udacity/fcnd-issue-reports#153]
At 1:39, i saw
Area = det([x1, x2, x3; y1, y2, y3; z1, z2, z3])

This is not true. The area is the length of the cross product of (v1 - v2) and (v1 - v3).

In this particular case, that the determinant is 0 just means the three points are collinear.

**Comment by lnson** [comment]
The same notation appears in Lesson 6.5.

**Comment by ryan-keenan** [comment]
Good catch! We updated the language in 6.5 a little while back to fix this, but the video in 6.4 still needs to be fixed.  That will be a slower process but we're on it!

**Comment by jtuckerdunn** [comment]
@kagure
